src/managers/gist-properties-manager.js

```diff
@@ -42,6 +43,10 @@ export function resolveMessageProperties(message) {
         {
             persistent = true
         }
+        if (message.properties.gist.exitClick)
```

Comment:
Do you ever think about pulling out the properties as a local variable? I ask because when I see a ton of references like this, the cognitive load of reading all of this can be lessened with some vars that skip reading a ton of dot notation.

Sometimes (don't know if this is still a JS issue) there is even a performance hit at scale with dot notation that you can shortcut with a local var, though a tradeoff in memory. (I don't think that matters in this case so don't consider this line here)